common-custom-user-data-scripts/gradle-common-custom-user-data.gradle

```diff
@@ -38,6 +38,9 @@ void tagOs() {
 void tagIde() {
     if (project.hasProperty('android.injected.invoked.from.ide')) {
         buildScan.tag 'Android Studio'
+        if (project.hasProperty('android.injected.studio.version')) {
+            buildScan.value 'AS Version', project.getProperty('android.injected.studio.version')
+        } 
```

Comment:
Given the existing tag is called `Android Studio`, I would call the custom value `Android Studio version`. Or we also rename the tag to `AS`. WDYT?

Reply:
That's a good point. The longer name isn't a problem anymore either since we can now edit tags. Updated.